contrib/fwupd.spec.in

```diff
@@ -424,118 +424,10 @@ done
 /usr/lib/udev/rules.d/*.rules
 /usr/lib/systemd/system-shutdown/fwupd.shutdown
 %dir %{_libdir}/fwupd-%{fwupdplugin_version}
```

Comment:
while spinning this whole series I think you should fix the instances of %{fwupdplugin_version} to just be %{fwupd_version}

Reply:
Ahh, I think it can be even simpler, and just use `%{version}` -- and we can make it .spec again not .spec.in IIRC

Reply:
I've pulled this to another branch we can look at after this one. The plugins change seemed to work fine, the innocuous sources splitout seems to have broken things. I'll split that out to a new PR too.